mongomock/aggregate.py

```diff
@@ -488,6 +490,39 @@ def _handle_type_convertion_operator(self, operator, values):
                 'You need to import the pymongo library to support decimal128 type.'
             )
 
+        # Document: https://docs.mongodb.com/manual/reference/operator/aggregation/toDecimal/
+        if operator == '$toDecimal':
+            if not decimal_support:
+                raise NotImplementedError(
```

Comment:
For coverage, please add a simple test for this.

Reply:
Sure thing. I have add it.